src/components/filterSection/FilterSection.js

```diff
@@ -0,0 +1,23 @@
+// components
+import SearchBox from '../../commonComponents/searchBox'
+import Checkbox from "../../commonComponents/checkbox/Checkbox";
```

Comment:
You already have and index file in checkbox folder no need for the last "Checkbox"